src/main/java/bar/Simplest2.java

```diff
@@ -4,6 +4,9 @@
 
 	public String bar;
 	
-	public static void foo() {}
+	// TODO bam
+	public static void foo() {
+		int a = 1;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused "a" local variable. ([squid:S1481](https://rebloch.sonarsource.com/organizations/fab/rules#rule_key=squid%3AS1481))

[See it in SonarCloud](https://rebloch.sonarsource.com/project/issues?id=test%3Afake-project-for-tests&pullRequest=1&issues=AWHcBRsyQYL-lH1H_cI_&open=AWHcBRsyQYL-lH1H_cI_)

[ISSUE_KEY]: AWHcBRsyQYL-lH1H_cI_